visualization.spectrum/resources/settings.xml

```diff
@@ -68,6 +69,46 @@
             <formatlabel>30018</formatlabel>
           </control>
         </setting>
+
+        <setting id="rotation_x" type="integer" label="30020">
+          <default>10</default>
+          <constraints>
+            <minimum>0</minimum>
+            <step>5</step>
+            <maximum>90</maximum>
+          </constraints>
+          <control type="slider" format="integer">
+            <formatlabel>30018</formatlabel>
+            <popup>false</popup>
+          </control>
+        </setting>
+
+        <setting id="offset_y" type="integer" label="30021">
+          <default>-25</default>
+          <constraints>
+            <minimum>-150</minimum>
+            <step>5</step>
+            <maximum>100</maximum>
+          </constraints>
+          <control type="slider" format="integer">
+            <formatlabel>30022</formatlabel>
+            <popup>false</popup>
+          </control>
+        </setting>
+
+        <setting id="field_size" type="integer" label="30023">
+          <default>100</default>
+          <constraints>
+            <minimum>50</minimum>
+            <step>5</step>
+            <maximum>150</maximum>
+          </constraints>
+          <control type="slider" format="integer">
+            <formatlabel>30024</formatlabel>
+            <popup>false</popup>
+          </control>
+        </setting>
```

Comment:
As this three new settings only works with GL must you add the:
```xml
          <dependencies>
            <dependency type="visible">
              <or>
                <condition on="property" name="IsDefined">HAS_GL</condition>
                <condition on="property" name="IsDefined">HAS_GLES</condition>
              </or>
            </dependency>
          </dependencies>
```
to prevent show on Windows and his DX.

Reply:
> As only part on view where seems to improved, is when no sound comes anymore, There stays one row still a bit up:

This was never seen here. If you paused, it is maybe normal behavior because shifting rows is paused too then.

> As this three new settings only works with GL must you add the:
>           <dependencies>
> to prevent show on Windows and his DX.

Done